template.json

```diff
@@ -218,6 +218,7 @@
     },
     "condition": {
       "condition" : {
+        "showName": false,
```

Comment:
Maybe I'm overlooking something but where is this value used?

Reply:
I might have jumped the gun here, but as I'm adding a new property to the condition property, I do also need to declare it here in the basic templates, no? I know that with JS, you can do a lot of stuff with objects, but I thought that it would make sense that if I'm going to use the new property, it should be also here.

Reply:
Right but it's a property of the modifier item, not the actor. I don't see anywhere where `actor.condition.showName` is used, or similar.

Reply:
Yeah, I definitely made a mishap here, I'll remove that! I had an initial idea to change all of them at the same time, prior to change it mod-by-mod, this seems to be a remnant of it.